src/utils/schema.js

```diff
@@ -54,45 +51,52 @@ module.exports.mergeMultiObjectFields = function (schema, objs) {
 	return model;
 };
 
-module.exports.slugifyFormID = function (schema, prefix = "") {
+const slugifyFormID = (schema, prefix = "") => {
 	// Try to get a reasonable default id from the schema,
 	// then slugify it.
 	if (typeof schema.id !== "undefined") {
 		// If an ID's been explicitly set, use it unchanged
 		return prefix + schema.id;
 	} else {
 		// Return the slugified version of either:
-		return prefix + (schema.inputName || schema.label || schema.model || "")
+		return (
+			prefix +
+			(schema.inputName || schema.label || schema.model || "")
+				// NB: This is a very simple, conservative, slugify function,
```

Comment:
we already rely on lodash, why not just use _.kebabCase() here so it's a bit clearer and shorter?

Reply:
Again, I'm sorry, I have no idea why we do that. This is a good suggestion but I'm not going to implement too much changes in this PR (there is already a lot of changes). Maybe next PR ? Or v3 ?

Reply:
Sounds good - we should probably do a full code review at some point and simplify whatever we can, relying on known functions with clear documentation (such as kebabCase).

Reply:
You are absolutely right !